### PR TITLE
Add compat data for NodeIterator

### PR DESCRIPTION
**Pull request by maboa**
https://developer.mozilla.org/en-US/docs/Web/API/NodeIterator

### CODE REVIEW

**Comment by ExE-Boss** [comment]
FYI, you added this PR to [`NodeFilter`](https://developer.mozilla.org/en-US/docs/Web/API/NodeFilter) and not to [`NodeIterator`](https://developer.mozilla.org/en-US/docs/Web/API/NodeIterator).

**Comment by maboa** [comment]
Thanks all :)

File: api/NodeIterator.json [hunk 1]
```diff
@@ -0,0 +1,488 @@
...
+            "version_added": "3"
+          }
+        },
+        "status": {
+          "experimental": false,
+          "standard_track": true,
+          "deprecated": false
+        }
+      },
+      "baseURI": {
```
Comment (Elchi3) [inline api/NodeIterator.json]: The key needs to be `expandEntityReferences` here

Comment (maboa) [reply api/NodeIterator.json]: OK - good spot - done!

**Pull request merged**